### ISSUE TITLE
Ubuntu Software Installer Not Detecting Version Changes

### ISSUE THREAD

**Issue by pivilartisant** [massalabs/station#1187]
**The problem(s)**

When updating MS version from old version to latest while following the Ubuntu GUI update flow, the Software Installer does not detect version changes. 

We should investigate to see if any improvements can be done here

**Comment by longgamma-massa** [comment]
@Thykof what are the implications of this? Users don't see the update button?

**Comment by Thykof** [comment]
I tested:
- uninstall MS
- open .dev of version v0.3.4, click on install, open MS
- click on the icon tray, click on "update now", confirm on the dialog, MS stops by itself and open github releases page
- open .deb of version v0.3.5, Ubuntu Software proposes me to remove MS, I remove MS, Ubuntu Software says "unable to find massastation". I close Ubuntu Software, I open .dep of version v0.3.5 with Ubuntu Software and install it normally.

**Comment by gregLibert** [comment]
I'm not sure will we have the ressource to tackle that soon. @SlnPons what about adding it to the troubleshooting guide instead and close this issue once it's done?

**Comment by SlnPons** [comment]
yes, @thomas-senechal could you add this to the troubleshooting guide please?
Or organise a catch up with me to sync on Ubuntu troubleshooting as I'm not 100% sure we had this scenario in mind last time we modify the guide.
